src/render/decorations/CHyprGroupBarDecoration.cpp

```diff
@@ -299,6 +301,8 @@ void refreshGroupBarGradients() {
     renderGradientTo(m_tGradientInactive, ((CGradientValueData*)PGROUPCOLINACTIVE->get())->m_vColors[0]);
     renderGradientTo(m_tGradientLockedActive, ((CGradientValueData*)PGROUPCOLACTIVELOCKED->get())->m_vColors[0]);
     renderGradientTo(m_tGradientLockedInactive, ((CGradientValueData*)PGROUPCOLINACTIVELOCKED->get())->m_vColors[0]);
+
+    g_pHyprRenderer->unsetEGL();
```

Comment:
not needed, and if called from within a rendering context will cause issues.

Reply:
done